metrics/web/test/common/presentation/metrics_theme/state/theme_notifier_test.dart

```diff
@@ -7,62 +8,112 @@ import 'package:test/test.dart';
 
 void main() {
   group("ThemeNotifier", () {
+    const lightTheme = ThemeType.light;
+    const darkTheme = ThemeType.dark;
+
     test(
-      "creates an instance with the default is dark value if the brightness is not specified",
+      "creates an instance with the light selected theme if the brightness is not specified",
       () {
         final themeNotifier = ThemeNotifier();
 
-        expect(themeNotifier.isDark, isNotNull);
+        expect(themeNotifier.selectedTheme, equals(lightTheme));
       },
     );
 
     test(
-      "creates an instance with the default is dark value if the given brightness is null",
+      "creates an instance with the light selected theme if the given brightness is null",
       () {
         final themeNotifier = ThemeNotifier(brightness: null);
 
-        expect(themeNotifier.isDark, isNotNull);
+        expect(themeNotifier.selectedTheme, lightTheme);
       },
     );
 
     test(
-      "creates an instance with the is dark value that corresponds to the given system's theme dark brightness",
+      "creates an instance with the dark selected theme that corresponds to the given system's theme dark brightness",
       () {
         final themeNotifier = ThemeNotifier(brightness: Brightness.dark);
 
-        expect(themeNotifier.isDark, isTrue);
+        expect(themeNotifier.selectedTheme, equals(darkTheme));
       },
     );
 
     test(
-      "creates an instance with the light theme mode that corresponds to the given light brightness",
+      "creates an instance with the light selected theme that corresponds to the given light brightness",
       () {
         final themeNotifier = ThemeNotifier(brightness: Brightness.light);
 
-        expect(themeNotifier.isDark, isFalse);
+        expect(themeNotifier.selectedTheme, equals(lightTheme));
+      },
+    );
+
+    test(".isDark is true if the selected theme is dark", () {
+      final themeNotifier = ThemeNotifier();
+      themeNotifier.changeTheme(darkTheme);
+
+      expect(themeNotifier.isDark, isTrue);
+    });
+
+    test(".isDark is false if the selected theme is light", () {
+      final themeNotifier = ThemeNotifier();
+      themeNotifier.changeTheme(lightTheme);
+
+      expect(themeNotifier.isDark, isFalse);
+    });
+
+    test(".changeTheme() changes the theme to the given value", () {
+      final themeNotifier = ThemeNotifier();
+      themeNotifier.changeTheme(lightTheme);
+
+      expect(themeNotifier.selectedTheme, equals(lightTheme));
+    });
+
+    test(
+      ".changeTheme() does not change the selected theme if the given value is null",
+      () {
+        final themeNotifier = ThemeNotifier();
+
+        expect(themeNotifier.selectedTheme, equals(lightTheme));
```

Comment:
Seems like this `expect` is redundant because we've already tested the default value of the `selectedTheme`.